host.yaml

```diff
@@ -7,11 +7,7 @@ repos:
 # - rhel-7.5-server-optional
 # - rhel-7.5-server-extras
 # - rhel-7.5-atomic
-  - dustymabe-ignition
```

Comment:
let's keep `dustymabe-ignition` repo for now. it has the latest up to date ignition and we don't necessarily want to wait for it to hit f29 stable before using it

Reply:
Umm, we do have latest ignition in f29. ostree repo which I compose locally using F29 content has ignition-0.27.0-3.gitcc7ebe0.fc29.x86_64. Also, [dustymabe copr repo](https://copr.fedorainfracloud.org/coprs/dustymabe/ignition/builds/) has same ignition as latest one. But, in case we are planning to do more updates in copr ignition, then I will definitely add dustymabe-ignition back.